tools/mkmapdata/mkmapdata.sh

```diff
@@ -527,7 +532,23 @@ until ((nfile>${#INGRID[*]})); do
       runcmd "ncatted -a history,global,a,c,"$history"  ${OUTFILE[nfile]}"
       runcmd "ncatted -a hostname,global,a,c,$HOST   -h ${OUTFILE[nfile]}"
       runcmd "ncatted -a logname,global,a,c,$LOGNAME -h ${OUTFILE[nfile]}"
```

Comment:
The following change was discussed in this post:
https://github.com/ESCOMP/CTSM/pull/823#issuecomment-582951237